Makefile

```diff
@@ -6,7 +6,7 @@ BUILDTAGS=
 RUNTIME ?= runc
 COMMIT ?= $(shell git describe --dirty --long --always --tags 2> /dev/null)
 VERSION := ${shell cat ./VERSION}
-BUILD_FLAGS := -tags "$(BUILDTAGS)" -ldflags "-X main.gitCommit=$(COMMIT) -X main.version=$(VERSION)" $(EXTRA_FLAGS)
+BUILD_FLAGS := -tags "$(BUILDTAGS)" -ldflags "-X main.gitCommit=$(COMMIT) -X main.version=$(VERSION) -buildmode=pie" $(EXTRA_FLAGS)
```

Comment:
The `-buildmode=pie` is not supported on mips platforms, but I guess no one cares about mips nowadays.